modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
+The following condition types are consistent across all ClusterVersion and ClusterOperator conditions. Individual operators may also have additional condition types with operator-specific language.
```

Comment:
Word choice for the second sentence: "Some" instead of "individual" in my opinion